tests/test_align.py

```diff
@@ -93,3 +93,94 @@ def test_astroquery(self):
         rms_y = max(shift_file['col7'])
 
         assert (rms_x <= 0.25 and rms_y <= 0.25)
+
+    def test_align_randomFields(self):
+        """ Process a large number of randomly selected fields which are defined in
+            an input ascii file (CSV).
+
+            Each field is used as input to determine if it can be aligned to an 
+            astrometric standard.  The success or fail status for each test is retained
+            as the overall success or fail statistic is the necessary output from
+            this test.
+        """
+
+        # Need to accommodate a pre-defined list existing in local store or Artifactory
```

Comment:
I don't think you need to -- I think it is okay if this function pulls everything from MAST